### PR TITLE
XCP-ng.Center 20.04.01

### PR DESCRIPTION
**Pull request by Geek-Almighty**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16891)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/16888.

**Comment by Geek-Almighty** [comment]
Submission containing materials of a third party: XCP-ng

File: manifests/x/XCP-ng/XCP-ngCenter20/04/01/20.04.01.33/XCP-ng.XCP-ngCenter20.04.01.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: XCP-ng.XCP-ngCenter20.04.01
```
Comment (Geek-Almighty) [inline manifests/x/XCP-ng/XCP-ngCenter20/04/01/20.04.01.33/XCP-ng.XCP-ngCenter20.04.01.installer.yaml:3]: Change to XCP-ng Center

**Review by Geek-Almighty — COMMENTED** [review]
Package renamed as suggested.

**Pull request closed without merging**